apps/opik-frontend/src/components/pages/PlaygroundPage/PlaygroundOutputs/PlaygroundOutputActions/PlaygroundOutputActions.tsx

```diff
@@ -448,6 +449,11 @@ const PlaygroundOutputActions = ({
 
   return (
     <>
+      {isRunning && datasetId && (
+        <div className="mb-4 mt-2">
+          <PlaygroundProgressIndicator />
```

Comment:
UX review:

Let's have it appear above the separator line, so it doesn't jump with progress bar:
1. After clicking play
2. When it finishes